src/main/java/core/basesyntax/figures/RightTriangle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.figures;
+
+public class RightTriangle extends Triangle {
```

Comment:
In my opinion is is better to think that Triangle is and extended RightTriangle, because in your implementation RightTriangle inherites 3 fields from Triangle , while only 2 is required. Also there are 6 fileds in RightTriangle  (3 from Triangle , 1 from Figure and 2 own), but only 2 is required.